docs/index.asciidoc

```diff
@@ -174,10 +164,10 @@ Example
     }
 
 
-NOTE: Starting with Logstash 6.0, the `document_type` option is
+NOTE: As of Logstash 6.0, the `document_type` option is
```

Comment:
We can remove this whole note block. document_type was never really a setting in this plugin, and we still support types in a few places in Logstash.